### PR TITLE
fix: serve self-hosted fonts

### PR DESCRIPTION
**Pull request by danroc**
This PR removes the fonts from the `static/` folder and serve them using the `webfonts` plugin.

### CODE REVIEW

**Comment by HowardBraham** [comment]
Are we actually using all of these font sub-types?  My browser downloads 15 woff2 files.

**Comment by danroc** [comment]
Indeed! I removed the variants that we are not using (directly at least).

**Comment by danroc** [comment]
@SocketSecurity ignore camelcase-css@2.0.1
@SocketSecurity ignore lodash.isempty@4.4.0
@SocketSecurity ignore proxy-from-env@1.1.0
@SocketSecurity ignore axios@1.5.0

File: packages/site/src/components/styledComponents.ts [hunk 1]
```diff
@@ -173,7 +172,7 @@ export const AccountRowTitle = styled.p`
   color: #000;
 
   /* H6 - Bold 14px */
-  font-family: Euclid Circular B;
+  font-family: Euclid Circular B, ui-sans-serif;
```
Comment (montelaidev) [inline packages/site/src/components/styledComponents.ts:175]: Is this valid?

**Review by montelaidev — APPROVED** [review]

File: packages/site/src/components/styledComponents.ts (hunk 1 above)
Comment (montelaidev) [reply packages/site/src/components/styledComponents.ts:175]: It’s the default system font and the naming is from the docs

**Pull request merged**